.github/workflows/tests.yml

```diff
@@ -92,6 +92,12 @@ jobs:
             pyqt6-ver: '!=6.6.0'
             # https://bugreports.qt.io/projects/PYSIDE/issues/PYSIDE-2346
             pyside6-ver: '!=6.5.1'
+          - os: ubuntu-24.04
+            python-version: '3.12'
+            # https://www.riverbankcomputing.com/pipermail/pyqt/2023-November/045606.html
+            pyqt6-ver: '!=6.6.0'
+            # https://bugreports.qt.io/projects/PYSIDE/issues/PYSIDE-2346
+            pyside6-ver: '!=6.5.1'
```

Comment:
I believe that many of these incompatible-version exclusions could be removed now; they do have some value to communicate that problems did occur, but there have been multiple subsequent releases of `pyqt6` and `pyside6` -- and perhaps also the other packages (I am yet to check the others).

Reply:
We can do this later. Given that the ubuntu-20.04 runner is about to expire, we should see that we get #29796 and this in.